src/patternfly/components/ModalBox/styles.scss

```diff
@@ -69,12 +155,50 @@
     overscroll-behavior: contain;
   }
 
+  // Footer
   &__footer {
+    display: flex;
     flex: 0 0 auto;
+    align-items: center;
     padding-top: var(--pf-c-modal-box__footer--PaddingTop);
     padding-right: var(--pf-c-modal-box__footer--PaddingRight);
     padding-bottom: var(--pf-c-modal-box__footer--PaddingBottom);
     padding-left: var(--pf-c-modal-box__footer--PaddingLeft);
+
+    // Using margin-left instead of justify: flex-end; as there may be other elements in the footer that align left
+    // Limit scope to direct child buttons, allows use of layouts if needed
+    > .pf-c-button:first-of-type {
+      margin-left: auto;
+    }
+
+    > .pf-c-button:not(:last-child) {
+      // Base margin left and right equal for buttons when wrapped
+      margin-right: calc(var(--pf-c-modal-box__footer--ButtonSpacer) / 2);
+
+      @media screen and (min-width: $pf-global--breakpoint--sm) {
+        margin-right: var(--pf-c-modal-box__footer--ButtonSpacer);
+      }
+    }
+  }
+
+  // Button close
+  &__button-close {
```

Comment:
why not use an action button here? It seems that it is the same minus the positioning?

Reply:
I'd expect the close button to also be the same as any other icon button.
The only concern I had brought up with related to the layout.